.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug
+about: Submit a bug report
+labels: bug, new
```

Comment:
We can either automatically add `new` (or similar) label to each new issue reported and then remove it as part of triage, or we can have all new issues have no label at all and the triager adds one.

I don't have strong opinions, but went with adding `new` here to everything.

Reply:
As someone with a terrible memory, I much prefer the idea of issues being auto-labelled and having that visual trigger to remove them (if necessary), rather than them being blank and having to remember to add one -- +1 from me

Reply:
+1 on more labels and redacting them manually as we triage.